src/IntegrationTests/java/uk/gov/hmcts/reform/sscs/config/DocumentConfigurationTest.java

```diff
@@ -39,13 +39,13 @@ public class DocumentConfigurationTest {
     public static Object[][] documentParameters() {
         return new Object[][] {
                 {LanguagePreference.ENGLISH, EventType.CORRECTION_GRANTED, Optional.empty(), "TB-SCS-GNO-ENG-corrected-decision-notice.docx"},
```

Comment:
should we get these from the yaml as well?

Reply:
maybe a task as part of clean up too?